tests/fv3_conf/gfs_v16_run.IN

```diff
@@ -19,6 +19,15 @@ cp    @[INPUTDATA_ROOT]/FV3_input_data/*_table .
 cp    @[INPUTDATA_ROOT]/FV3_input_data/diag_table_gfdlmp diag_table
 cp    @[INPUTDATA_ROOT]/FV3_input_data_gsd/field_table_suite2 field_table
 cp    @[INPUTDATA_ROOT]/FV3_input_data/*configure .
+cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.*nc .
```

Comment:
We have several gfs_v16 tests using this gfs_v16_run.IN, do you need to update those tests too to use IAER=1111? The IAER needs to be defined for the following test too.
fv3_ccpp_gfs_v16_RRTMGP, fv3_ccpp_gfs_v16_RRTMGP_debug, fv3_ccpp_gfs_v16_debug, fv3_ccpp_gfs_v16_restart, fv3_ccpp_gfs_v16_stochy